eras/shelley/impl/src/Cardano/Ledger/Shelley/API/ByronTranslation.hs

```diff
@@ -148,7 +148,8 @@ translateToShelleyLedgerState genesisShelley epochNo cvs =
               { _utxo = utxoShelley,
                 _deposited = Coin 0,
                 _fees = Coin 0,
-                _ppups = def
+                _ppups = def,
+                _stakeDistro = IStake mempty Map.empty
```

Comment:
I think this is wrong, when we translate from Byron, the initial incremental stake needs to be
```
aggregateUtxoCoinByCredential Map.empty utxoShelley
```

We can use `Map.empty` for the pointer map since there will be no registered stake credentials at the start of Shelley.